src/lexedata/exporter/coverage.py

```diff
@@ -20,10 +20,11 @@
         help="Only list matching languages, don't report statistics",
     )
     parser.add_argument(
-        "--dataset",
-        default="Wordlist-metadata.json",
+        "--metadata",
+        nargs="?",
         type=Path,
-        help="Metadata file or forms.csv file (i.e. metadata-free wordlist) to inspect",
+        default="Wordlist-metadata.json",
+        help="Path to the metadata.json. The metadata file describes the dataset. Default: ./Wordlist-metadata.json",
```

Comment:
Here we have an example of a script that should be able to run metadata-free, i.e. on a bare `forms.csv` file. How do you suggest we treat this on the CLI? What is a good way to for the user when typing `program -h` to see whether a program takes metadata only, or also metadata-free `forms.csv`s?

I would suggest we use a switch `--metadata-or-forms` with explicit alias `--metadata` (it's an implicit alias already, because you can always shorten non-ambiguous switches) and add the `or forms.csv file (i.e. metadata-free wordlist)` if that is the case, if that looks okay on the CLI help display.